apps/charterafrica/src/pages/index.page.js

```diff
@@ -855,6 +855,74 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
           href: "/",
           priority: true,
         },
+        menus: [
+          {
+            title: "Resources",
+            href: "resources",
+            children: [
+              {
+                title: "Tools",
+                href: "resources/tools",
+              },
+              {
+                title: "Data",
+                href: "resources/data",
+              },
+              {
+                title: "People",
+                href: "resources/people",
+              },
+            ],
+          },
+          {
+            title: "Knowledge",
+            href: "knowledge",
+            children: [
+              {
+                title: "Explainers",
+                href: "knowledge/explainers",
+              },
+              {
+                title: "News",
+                href: "knowledge/news",
+              },
+              {
+                title: "Research",
+                href: "knowledge/research",
+              },
+              {
+                title: "Academy",
+                href: "knowledge/academy",
+              },
+            ],
+          },
+          {
+            title: "Oppportunities",
+            href: "opportunities",
+            children: [
+              {
+                title: "Grants and Fellowships",
+                href: "opportunities/grants-and-fellowships",
+              },
+              {
+                title: "Events",
+                href: "opportunities/events",
+              },
+              {
+                title: "Community",
+                href: "opportunities/community",
+              },
+              {
+                title: "Democracy Helpdesk",
+                href: "opportunities/democracy-helpdesk",
+              },
+            ],
+          },
+          {
+            title: "About",
+            href: "about",
```

Comment:
```suggestion
          {
            title: "Resources",
            href: "/",
            children: [
              {
                title: "Tools",
                href: "/",
              },
              {
                title: "Data",
                href: "/",
              },
              {
                title: "People",
                href: "/",
              },
            ],
          },
          {
            title: "Knowledge",
            href: "/",
            children: [
              {
                title: "Explainers",
                href: "/",
              },
              {
                title: "News",
                href: "/",
              },
              {
                title: "Research",
                href: "/",
              },
              {
                title: "Academy",
                href: "/",
              },
            ],
          },
          {
            title: "Oppportunities",
            href: "/",
            children: [
              {
                title: "Grants and Fellowships",
                href: "/",
              },
              {
                title: "Events",
                href: "/",
              },
              {
                title: "Community",
                href: "/",
              },
              {
                title: "Democracy Helpdesk",
                href: "/",
              },
            ],
          },
          {
            title: "About",
            href: "/",
```

Since we don't have child pages, all links should just point to `"/"`.